subcommands/common_config.go

```diff
@@ -93,6 +102,11 @@ func LogConfigs(opts *LogConfigsOptions) {
 		}
 		DieNotNil(err)
 		for _, cfg := range dcl.Configs {
+			if len(cfg.CreatedBy) > 0 {
+				if v, ok := opts.UserLookup[cfg.CreatedBy]; ok {
+					cfg.CreatedBy = fmt.Sprintf("%s / %s", v.PolisId, v.Name)
+				}
```

Comment:
Maybe, show `deadbeef / <deleted user>` or `deadbeef / <unknown user>` when there is no information in a map.
It looks like we are losing some information _as is_.

Reply:
I *think* (but couldn't confirm with a quick search) that we do a "?" when the username isn't found.